.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+os:
+  - linux
+services:
+  - redis
+node_js:
+  - "0.10"
+  - "0.12"
+  - "4.1"
```

Comment:
Any reason to pin to 4.1.x here instead of just 4.x (i.e. `- "4"`)?

Also, any reason to test against 0.10 and 0.12? All newww deployments should now be running Node 4.